core/src/main/java/org/wildfly/extras/creaper/core/online/OnlineManagementClientImpl.java

```diff
@@ -149,7 +149,8 @@ public void apply(Iterable<OnlineCommand> commands) throws CommandFailedExceptio
     public ModelNodeResult execute(ModelNode operation) throws IOException {
         checkClosed();
         operation = adjustOperationForDomain.adjust(operation);
-        log.debugf("Executing operation %s", ModelNodeOperationToCliString.convert(operation));
+        log.debugf("Executing operation %s\nJSON format:\n%s", ModelNodeOperationToCliString.convert(operation),
+                operation.toJSONString(false));
```

Comment:
BTW, `Operation.getOperation` gives you a `ModelNode`, but the entire purpose of the `Operation` class is to have attachments, the `ModelNode` doesn't contain everything. Just a notice.

I'm still not exactly fond of expanding a single-line log message to 3 lines, but I guess I can be convinced. Keep on pushing :-)